### ISSUE TITLE
Ability to configure filter for entrance/home page

### ISSUE THREAD

**Issue by cudor** [MISP/MISP#4291]
It would be great if each user had the possibility to configure a filter for the events listed on the "home" page displayed after logging in. The intention is to have a cleaner list of events that are most relevant to the respective analyst.

Filtering should be possible based on tags (on event level) and organisations. Other potential filter conditions might be whether events are published or not, on the threat level, state of the analysis, ...

**Comment by iglocska** [comment]
Indeed. The intent is to have this at some point, the biggest hurdle is that we need a user setting system - currently it's on the back burner but we have more and more use-cases for it.

**Comment by nbareil** [comment]
For the record and context, that's probably linked to #2005
